src/osg/Shader.cpp

```diff
@@ -407,36 +407,34 @@ Shader::ShaderObjects::ShaderObjects(const osg::Shader* shader, unsigned int con
 
```

Comment:
Missing faster shader define stuff at the top. Probably newer change than the last time makOsg was synced with this branch.